templates/todo/projects/csharp-cosmos-sql/.repo/bicep/infra/main.bicep

```diff
@@ -154,6 +186,7 @@ output AZURE_KEY_VAULT_ENDPOINT string = keyVault.outputs.endpoint
 output AZURE_KEY_VAULT_NAME string = keyVault.outputs.name
 output AZURE_LOCATION string = location
 output AZURE_TENANT_ID string = tenant().tenantId
-output REACT_APP_API_BASE_URL string = api.outputs.SERVICE_API_URI
+output REACT_APP_API_BASE_URL string = (useAPIM == 'true') ? apimApi.outputs.SERVICE_API_URI : api.outputs.SERVICE_API_URI
 output REACT_APP_APPLICATIONINSIGHTS_CONNECTION_STRING string = monitoring.outputs.applicationInsightsConnectionString
 output REACT_APP_WEB_BASE_URL string = web.outputs.SERVICE_WEB_URI
+output USE_APIM string = useAPIM
```

Comment:
@vieiraae I don't see any immediate usage of `USE_APIM` as an output parameter (which is largely for local development)? Do we need this? Otherwise, I suggest removing it

Reply:
That is so easy for someone to edit if they open the file.

Reply:
@jongio Do you think it's worth doing? I feel that it pollutes the environment file when it's not enabled. They would be able to edit it simply by updating `parameters.json`, or realizing that `parameters.json` references an env variable, and setting the env variable.

Setting the value in environment file to then affect the next provision seems great for demo/local, but wouldn't actually work in production. I could still see it useful for demo scenarios, but it also makes it less understandable from a non-demo scenario.

Reply:
Yeah, we can just keep it, doesn't hurt IMO.